### PR TITLE
AN-5523 tags

### PR DESCRIPTION
**Pull request by eric-laurello**

### CODE REVIEW

File: models/silver/core/silver___all_undecoded_instructions_data.sql
Comment (tarikceric) [inline models/silver/core/silver___all_undecoded_instructions_data.sql:1]: super old model that isnt even used anymore - can be moved to noncore

File: models/silver/core/silver___blocks_tx_count.sql
Comment (tarikceric) [inline models/silver/core/silver___blocks_tx_count.sql:1]: doesnt run on core schedule

File: models/silver/core/silver__epoch.sql
Comment (tarikceric) [inline models/silver/core/silver__epoch.sql:1]: doesnt run on core schedule - move to noncore

**Comment by tarikceric** [comment]
Theres also `models/silver/program_logs/silver__transaction_logs_program_data.sql` which runs on core job, but its in the `silver/program_logs` folder so should that folder be moved to `silver/core`?

**Comment by eric-laurello** [comment]
> Theres also `models/silver/program_logs/silver__transaction_logs_program_data.sql` which runs on core job, but its in the `silver/program_logs` folder so should that folder be moved to `silver/core`?

I can just add the folder to the core model list

File: models/silver/core/silver___blocks_tx_count.sql
Comment (tarikceric) [reply models/silver/core/silver___blocks_tx_count.sql:1]: mightve accidently copied it over instead of moving

Comment (eric-laurello) [reply models/silver/core/silver___blocks_tx_count.sql:1]: updated, thanks for catching

**Review by tarikceric — APPROVED** [review]

**Pull request merged**